gobblin-runtime/src/main/java/org/apache/gobblin/scheduler/JobScheduler.java

```diff
@@ -583,12 +582,17 @@ public void close() throws IOException {
   }
 
   /**
-   * Get a {@link org.quartz.Trigger} from the given job configuration properties.
+   * Get a {@link org.quartz.Trigger} from the given job configuration properties. If triggerSuffix is provided, appends
+   * it to the end of the flow name.
    */
-  public static Trigger createTriggerForJob(JobKey jobKey, Properties jobProps) {
```

Comment:
Did you verify that this method is not called anywhere else? Asking because this is a backward incompatible change. You can leave this method and make it by default a call to createTriggerForJob(jobKey, jobProps, Optional.absent())

Reply:
Yea I checked everywhere and added. It would not compile otherwise also.